### PR TITLE
sysprof 47.0 (new formula)

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Needed for the latest version of webkitgtk (2.46.0).

See #190946.

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]

**Comment by carlocab** [comment]
Needs #190971. What a yak-shave...

File: Formula/s/sysprof.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  depends_on "libunwind"
+  depends_on :linux
+
+  def install
+    system "meson", "setup", "build",
+                    "-Dgtk=true",
+                    "-Dtools=true",
+                    "-Dtests=false",
+                    "-Dexamples=false",
+                    "-Dpolkit-agent=disabled", # requires dependencies not packaged yet
```
Comment (carlocab) [inline Formula/s/sysprof.rb:40]: Seems to still need `polkit` anyway. Annoying

**Comment by bevanjkay** [comment]
```
[322/384] /home/linuxbrew/.linuxbrew/opt/meson/bin/meson --internal msgfmthelper --msgfmt=/home/linuxbrew/.linuxbrew/opt/gettext/bin/msgfmt ../src/sysprofd/org.gnome.sysprof3.policy.in src/sysprofd/org.gnome.sysprof3.policy xml /tmp/sysprof-20241001-18056-h8c0vw/sysprof-47.0/po
  FAILED: src/sysprofd/org.gnome.sysprof3.policy 
  /home/linuxbrew/.linuxbrew/opt/meson/bin/meson --internal msgfmthelper --msgfmt=/home/linuxbrew/.linuxbrew/opt/gettext/bin/msgfmt ../src/sysprofd/org.gnome.sysprof3.policy.in src/sysprofd/org.gnome.sysprof3.policy xml /tmp/sysprof-20241001-18056-h8c0vw/sysprof-47.0/po
  /home/linuxbrew/.linuxbrew/opt/gettext/bin/msgfmt: cannot locate ITS rules for ../src/sysprofd/org.gnome.sysprof3.policy.in
```

**Comment by carlocab** [comment]
Note: the `test` block likely doesn't work yet. Probably needs `CFLAGS`, `LDFLAGS` and/or `LDLIBS`.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**